server/db/driver/pg/internal/matches.go

```diff
@@ -0,0 +1,43 @@
+package internal
+
+const (
+	CreateMatchesTable = `CREATE TABLE IF NOT EXISTS %s (
+		matchid BYTEA PRIMARY KEY,
+		takerOrder BYTEA,      -- INDEX this
+		takerAccount BYTEA,    -- INDEX this
```

Comment:
I missed `auth.Storage.ActiveMatches`.  Thanks for pointing that out.

Since `swap.Storage` presently provides `UpdateMatch(match *order.Match) error` to store and update matches, we'd need to update `order.Match` (and `order.MatchSet`) with the addresses as well.  However, because these structs see much of their use before the addresses are even solicited from the parties, those fields will be empty, to be populated during the swap.  As such, I'm inclined to change the methods in `swap.Storage` to avoid modifying `order.Match` and `order.MatchSet`  (e.g. add to `swap.Storage` a `NewMatch` function that takes the addresses too).

Reply:
Um, I don't know what I was smoking yesterday when I clearly forgot that the incoming `Order`s have the swap addresses. :man_facepalming: 
Anyway, the DB now stores the maker and taker addresses in the matches table too.